package-lock.json

```diff
@@ -27226,7 +27174,6 @@
       "version": "2.0.2",
       "resolved": "https://registry.npmjs.org/which/-/which-2.0.2.tgz",
       "integrity": "sha512-BLI3Tl1TW3Pvl70l3yq3Y64i+awpwXqsGBYWkkqMtnbXgrMD+yj7rhW0kuEDxzJaYXGjEW5ogapKNMEKNMjibA==",
-      "dev": true,
```

Comment:
Just to be sure, should all these `"dev": true` be removed, i.e. should all these modules be installed in production?

Reply:
this was automatically updated by npm...

Reply:
I see.. I am not sure we should merge the removal of those :s

Reply:
fixed! :)